My Project/UtilsCommon.vb

```diff
@@ -1513,7 +1513,8 @@ Public Class UtilsCommon
 
                     DocValues.Add(tmpValue)
                 Else
-                    Throw New Exception(String.Format("Property '{0}' not found", PropertyName))
+                    Throw New Exception(String.Format("Property '{0}' not found", PropertyName)) '<--- Instead of Throw an exception the situation should be handled. Throw an exception have an impact on performance
```

Comment:
Throw an exception when it can be handled have an impact on performance, specially in large lists.
In my opinion this should be changed and perhaps return a fixed value that can handle the situation.
I did some tests and in this case it seem to not have a large impact than other Try\Catch